index.html

```diff
@@ -475,19 +475,6 @@ <h2 class="icon icon-mailing-lists">Mailing Lists</h2>
 
       <div class="corner"></div>
 
-      <br/>
-
-      <div class="list-header">
-        <a href="https://mail-archives.apache.org/mod_mbox/couchdb-announce/"><strong>Mail Search</strong></a>
-        <p>Search all of our mailing lists at once.</p>
-      </div>
```

Comment:
https://lists.apache.org/list.html?announce@couchdb.apache.org is still there it seems?

Reply:
1. It's a much more confusing entry without the form following it.
2. The link is a lie (it only searches the announce list)

Reply:
Ah, you're right. Good point.